packages/eslint-config-base/src/rules/best-practices.ts

```diff
@@ -16,26 +13,13 @@ const config: Linter.Config = {
     // https://eslint.org/docs/rules/block-scoped-var
     'block-scoped-var': 'error',
 
-    // specify the maximum cyclomatic complexity allowed in a program
-    // https://eslint.org/docs/rules/complexity
-    complexity: ['off', 20],
-
-    // enforce that class methods use "this"
-    // https://eslint.org/docs/rules/class-methods-use-this
-    'class-methods-use-this': [
-      'error',
-      {
-        exceptMethods: [],
-      },
-    ],
-
     // require return statements to either always or never specify values
     // https://eslint.org/docs/rules/consistent-return
     'consistent-return': 'error',
 
     // specify curly brace conventions for all control statements
     // https://eslint.org/docs/rules/curly
-    curly: ['error', 'multi-line'], // multiline
+    curly: ['error', 'all'],
```

Comment:
Will this result in detecting more errors in the existing code checks?

Reply:
No. This rule remains the same prior to this PR.

This rule is previously overridden by us: https://github.com/RightCapitalHQ/frontend-style-guide/pull/166/files#diff-2c866a6ebba9015a6180c332af9b6f11eda5c1b1089cad4732a5625e11e65cf6L31-L32.

We always use `['error', 'all']` for this rule.

Additionaly, the `multi-line` setting conflicts with Prettier. Therefore, `eslint-config-prettier` disables the `curly` rule and leave it to the users manually specify a safe option: https://github.com/prettier/eslint-config-prettier?tab=readme-ov-file#curly